### PR TITLE
Rename BitTypography to BitText (#8465)

### LINKED ISSUE #8465: The `BitTypography` component needs to be renamed to `BitText`
The `BitTypography` component needs to be renamed to `BitText`

### PR DESCRIPTION
**Pull request by hootanht**
closes #8465

### CODE REVIEW

File: src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Bit.BlazorUI.Demo.Client.Core.csproj
Comment (msynk) [inline src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Bit.BlazorUI.Demo.Client.Core.csproj:1]: plz revert changes to this file

File: src/Templates/BlazorEmpty/Bit.BlazorEmpty/BlazorEmpty/Components/Pages/About.razor
Comment (msynk) [inline src/Templates/BlazorEmpty/Bit.BlazorEmpty/BlazorEmpty/Components/Pages/About.razor:1]: plz revert the changes in this file. you're not supposed to update other projects in this PR.

File: src/Templates/BlazorEmpty/Bit.BlazorEmpty/BlazorEmpty/Components/Pages/Home.razor
Comment (msynk) [inline src/Templates/BlazorEmpty/Bit.BlazorEmpty/BlazorEmpty/Components/Pages/Home.razor:1]: plz revert the changes in this file. you're not supposed to update other projects in this PR.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Templates/BlazorEmpty/Bit.BlazorEmpty/BlazorEmpty/Components/Pages/About.razor
Comment (hootanht) [reply src/Templates/BlazorEmpty/Bit.BlazorEmpty/BlazorEmpty/Components/Pages/About.razor:1]: If I revert the changes, the build will fail. However, if you mean that `BitTypography` won't be deleted and you'll just be making changes to another component, I'm okay with proceeding.

Comment (msynk) [reply src/Templates/BlazorEmpty/Bit.BlazorEmpty/BlazorEmpty/Components/Pages/About.razor:1]: hmm, what you're saying is incorrect, since this project uses the nuget package as the reference and does not use the BlazorUI source code directly.

Comment (hootanht) [reply src/Templates/BlazorEmpty/Bit.BlazorEmpty/BlazorEmpty/Components/Pages/About.razor:1]: Great!

File: src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Bit.BlazorUI.Demo.Client.Core.csproj
Comment (msynk) [inline src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Bit.BlazorUI.Demo.Client.Core.csproj:1]: you still have changes to this file. plz revert any changes in this file

Comment (msynk) [inline src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Bit.BlazorUI.Demo.Client.Core.csproj:1]: you made it even worse. plz revert all these so this file won't be in this PR.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Bit.BlazorUI.Demo.Client.Core.csproj
Comment (hootanht) [reply src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Bit.BlazorUI.Demo.Client.Core.csproj:1]: Done

**Review by msynk — APPROVED** [review]
